addons/website/static/src/js/website.editor.js

```diff
@@ -216,7 +216,7 @@ widget.LinkDialog.include({
 /**
  * Display a welcome message on the homepage when it is empty and that the user is connected.
  */
-base.ready().then(function () {
+$.when(def, base.ready()).then(function () {
```

Comment:
I'm looking for this but, they are a bug when we load this template on the website.